libmamba/src/core/context.cpp

```diff
@@ -179,6 +179,11 @@ namespace mamba
         return m_authentication_info;
     }
 
+    const std::map<std::string, AuthenticationInfo>& Context::authentication_info() const
```

Comment:
I think it would help future maintenance (and switching to unordered_map) if this map was a type alias.